### PR TITLE
Allow Indexing for Data Registry Files

### PR DESCRIPTION
**Pull request by sagehen03**

### CODE REVIEW

File: handler.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def main(event, context):
     # set environment variables
     os.environ['BIOINDEX_S3_BUCKET'] = s3_bucket
     os.environ['BIOINDEX_RDS_SECRET'] = rds_secret
-    os.environ['BIOINDEX_RDS_SCHEMA'] = rds_schema
+    os.environ['BIOINDEX_BIO_SCHEMA'] = rds_schema
```
Comment (sagehen03) [inline handler.py:38]: Bioindex uses BIOINDEX_BIO_SCHEMA to determine db schema to use. https://github.com/broadinstitute/dig-bioindex/blob/c9b668cee6197b1a86c76ad73557d75bd597c601/bioindex/lib/config.py#L174-L177

File: package-lock.json
Comment (sagehen03) [inline package-lock.json:1]: I had to run npm install before running `serverless deploy` and that step updated package-lock.json.

File: requirements.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
-git+git://github.com/broadinstitute/dig-bioindex.git@master#egg=bioindex
+git+https://github.com/broadinstitute/dig-bioindex.git@master#egg=bioindex
```
Comment (sagehen03) [inline requirements.txt:1]: I'm not sure if git stopped supporting the old protocol syntax or what.  But serverless could not find the bioindex dep without this.

File: requirements.txt [hunk 3]
```diff
@@ -1,2 +1,3 @@
...
 pymysql
+sqlalchemy==1.4
```
Comment (sagehen03) [inline requirements.txt:3]: It seems like without specifying the version, serverless bundles version 2.x which is not compatible with the bioindex :shrug:

File: requirements.txt (hunk 2 above)
Comment (psmadbec) [reply requirements.txt:1]: I would guess that this is just something odd with whatever weird key it was set up to use. I thought it was the https version that is being deprecated in the future, not the git one

File: requirements.txt (hunk 3 above)
Comment (psmadbec) [reply requirements.txt:3]: Yeah it won't be yet, although I probably have that branch which is like 50% of the way there I imagine (sans testing)

**Review by psmadbec — APPROVED** [review]
This all seems fine

**Pull request merged**